Automattic/themes
# Spearhead: adds customizer notice about dark mode.

This PR addresses the first part of #2642.

<img width="1904" alt="Screen Shot 2020-10-29 at 2 08 46 PM" src="https://user-images.githubusercontent.com/5375500/97615120-4461b980-19f1-11eb-98ca-7b0f2736bfe8.png">

**To Test**
- Check out this PR onto your sandbox
- Go to the customizer
- Verify the notice appears as expected
